chaindexing-core/src/events.ts

```diff
@@ -1,14 +1,21 @@
-export interface Event {
+export interface UnsavedEvent {
```

Comment:
What I meant was: `Event` is expected to be whole, while `UnsavedEvent` stays partial (without the id)

```suggestion
export interface Event {
  id: string
```

Reply:
This is resolved.